make/config.mk

```diff
@@ -107,7 +107,15 @@ USE_NGRAPH_DISTRIBUTED = 0
 # e.g. NGRAPH_DIR = <path>/ngraph/install (containing lib and include dirs)
 # if not specified nGraph will be cloned/built/installed during make
 NGRAPH_DIR =
-
+# If USE_NGRAPH=1 and NGRAPH_DIR is undefined/empty, then the following variables
+# are used to configure/build a local copy of nGraph:
+NGRAPH_EXTRA_MAKE_FLAGS="VERBOSE=1"
+NGRAPH_EXTRA_CMAKE_FLAGS="-DNGRAPH_USE_PREBUILT_LLVM=1"
```

Comment:
This isn't valid on some systems (Mac, Centos). I guess we need more complicated logic here?

Reply:
AFAIK the only currently supported platform for nGraph is Ubuntu 16.04.  Is that incorrect?

Reply:
We also officially support Centos 7 and Mac is on the priority list this quarter.